JavaScript internationalisation is broken

When I try to extract string from the UI code, I get this error: 

```
➜  ui git:(master) ✗ npm run translate

> aleph-ui@3.3.8 translate /Users/pudo/Code/aleph/ui
> npm run messages && npm run json2pot && npm run po2json


> aleph-ui@3.3.8 messages /Users/pudo/Code/aleph/ui
> react-intl-cra 'src/**/*.{js,jsx}' -o i18n/messages.json

sh: react-intl-cra: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! aleph-ui@3.3.8 messages: `react-intl-cra 'src/**/*.{js,jsx}' -o i18n/messages.json`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the aleph-ui@3.3.8 messages script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pudo/.npm/_logs/2019-12-03T20_15_59_760Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aleph-ui@3.3.8 translate: `npm run messages && npm run json2pot && npm run po2json`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the aleph-ui@3.3.8 translate script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

--- a/ui/package.json
+++ b/ui/package.json
@@ -38,6 +38,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-decorators": "^7.6.0",
+    "@formatjs/cli": "^1.1.18",
     "@svgr/webpack": "4.3.3",
     "babel-eslint": "^10.0.3",
     "babel-jest": "24.9.0",
@@ -98,7 +99,7 @@
   ],
   "scripts": {
     "translate": "npm run messages && npm run json2pot && npm run po2json",
-    "messages": "react-intl-cra 'src/**/*.{js,jsx}' -o i18n/messages.json",
+    "messages": "formatjs extract --out-file i18n/messages.json 'src/**/*.{js,jsx}'",
     "json2pot": "react-intl-po json2pot 'i18n/messages.json' -o 'i18n/messages.pot'",
     "po2json": "react-intl-po po2json 'i18n/translations/*.po' -m 'i18n/messages.json' -o src/content/translations.json",
     "start": "node scripts/start.js",

--- a/ui/package.json
+++ b/ui/package.json
@@ -76,7 +76,6 @@
     "postcss-safe-parser": "^4.0.1",
     "react-app-polyfill": "^1.0.4",
     "react-dev-utils": "^10.0.0",
-    "react-intl-cra": "^0.3.4",
     "react-intl-po": "^2.2.2",
     "redux-logger": "^3.0.6",
     "resolve": "^1.10.0",

